src/c++/perf_analyzer/genai-perf/genai_perf/export_data/console_exporter.py

```diff
@@ -36,74 +38,69 @@ class ConsoleExporter:
     A class to export the statistics and arg values to the console.
     """
 
+    STAT_COLUMNS = ["avg", "min", "max", "p99", "p90", "p75"]
```

Comment:
STAT_COLUMN_KEYS?

Reply:
Maybe `STATS_COLUMN_HEADER`?

Reply:
It looks like this is being used in for loops predominantly, so I'd probably keep it plural.

FWIW, for me, `STAT_COLUMN_KEYS` is clearer since this is a list of values. A header to me would be more like `STATS_COLUMN_HEADER = "avg|min|max|p99|p75" which isn't what it looks like you're going for here. If you don't like the work keys though, there are probably others like headings and labels.

Reply:
i had headers initially as my suggestion but moved away from it for the exact reason @dyastremsky mentioned above.